src/modules/alerting.ts

```diff
@@ -82,6 +89,17 @@ class AlertingApi extends BaseApi {
      * Error toasts (`type: 'error'`) require manual dismissal and must be explictly specified with `showCloseButton: true`.
      * TypeScript users will also specify `timeout: 0` while JavaScript users can specify 0 or omit the prop entirely.
      * The `timeout` prop will be ignored regardless.
+     * 
+     * **Toast Options:**
+     * 
+     * Name | Type | Default | Description |
```

Comment:
is the existence of a Default column enough to convey that these props are optional?

Reply:
looking at the few optional params we have, it does seem like the default value is an indicator of optional:

![image](https://github.com/MyPureCloud/client-app-sdk/assets/11987207/db5f6fc3-a239-47d1-9d38-659b3544f55e)
![image](https://github.com/MyPureCloud/client-app-sdk/assets/11987207/3a850717-77b0-4703-ba96-735bc5694e6e)

So, probably ok to have the column and ensure the default is specified or we put `-` or `none` for required attrs.  Otherwise, g2g as is and dont think we need a ton more to flag optional